.cirrus.yml

```diff
@@ -121,7 +121,7 @@ task:
         cpu: 4
         memory: 12G
     - name: tool_coverage-linux
-      skip: "!changesInclude('packages/flutter_tools/**/*.dart')"
+      skip: "!changesInclude('packages/flutter_tools/**/*.dart') || $CIRRUS_BRANCH == 'master'"
```

Comment:
@jonahwilliams, did you mean to add `&& $CIRRUS_BRANCH != 'master'`?

Reply:
D'oh